tooling/maven/sync-properties-maven-plugin/src/main/resources/camel-dependencies-template.xml

```diff
@@ -34,32 +34,23 @@
 
     <packaging>pom</packaging>
     <name>Camel :: Dependencies</name>
-    <description>Camel Dependencies POM</description>
+    <description>Version properties used for building and testing Apache Camel</description>
 
-    <!-- sync properties here from parent/pom.xml -->
     <properties>
         @properties@
     </properties>
 
-    <profiles>
-        <profile>
-            <id>full</id>
-            <activation>
-                <property>
-                    <name>!quickly</name>
-                </property>
-            </activation>
-            <build>
-                <pluginManagement>
-                    <plugins>
-                        <plugin>
-                            <groupId>com.mycila</groupId>
-                            <artifactId>license-maven-plugin</artifactId>
-                            <version>${mycila-license-version}</version>
-                        </plugin>
-                    </plugins>
-                </pluginManagement>
-            </build>
-        </profile>
-    </profiles>
+    <url>https://camel.apache.org</url>
+    <inceptionYear>2007</inceptionYear>
+    <scm>
+        <connection>scm:git:http://gitbox.apache.org/repos/asf/camel.git</connection>
+        <developerConnection>scm:git:https://gitbox.apache.org/repos/asf/camel.git</developerConnection>
+        <url>https://gitbox.apache.org/repos/asf?p=camel.git;a=summary</url>
```

Comment:
Are they still actual?

Reply:
gitbox redirects on github now, I think they are trying to find a fix in infra, but it's not done yet.

Reply:
I copied them from the top module without too much thinking. Given that I was able to cut a fake release, I assume these URLs really work.